source/retryable-reads/tests/unified/handshakeError.yml

```diff
@@ -0,0 +1,71 @@
+description: "retryable reads handshake failures"
+
+schemaVersion: "1.6"
```

Comment:
Is schemaVersion 1.6 actually require here or can we use a lower version? I think the best practice is to set this to the lowest version possible.

Reply:
done!